Info-Orbs/include/widgets/weatherWidget.h

```diff
@@ -44,7 +44,7 @@ class WeatherWidget : public Widget {
 
     WeatherDataModel model;
 
-    String weatherLocation = WEATHER_LOCAION;
+    String weatherLocation = WEATHER_LOCATION;
```

Comment:
There should be an ifdef in place to use the old string from someone's existing config.h